mypy/fastparse.py

```diff
@@ -398,7 +397,7 @@ def translate_type_comment(self,
         ast3.BitXor: '^',
         ast3.BitAnd: '&',
         ast3.FloorDiv: '//'
-    }  # type: Final[Dict[typing.Type[AST], str]]
+    }  # type: Final
```

Comment:
Let's keep this since otherwise it may be unclear what the inferred type would be.